docs/getting-started.md

```diff
@@ -15,7 +15,17 @@ The interactive course with quizzes will guide you through everything you need t
 - [Node.js 10](https://nodejs.org/) or later
 - MacOS, Windows (including WSL), and Linux are supported
 
-## Setup
+## Guided Setup
```

Comment:
This can probably just be "Setup" to keep anchors working.
```suggestion
## Setup
```

Reply:
Agree 👍